govc/USAGE.md

```diff
@@ -227,6 +227,11 @@ but appear via `govc $cmd -h`:
  - [metric.ls](#metricls)
  - [metric.reset](#metricreset)
  - [metric.sample](#metricsample)
+ - [modules.add_vms](#modulesadd_vms)
```

Comment:
Thinking we should put this under `cluster` and use singular `module`:
```
cluster.module.vm.add
cluster.module.create
cluster.module.ls
cluster.module.rm
```

For `cluster.module.vm`, we could do:
```
cluster.module.vm.add
cluster.module.vm.rm
cluster.module.vm.ls
```
Or even:
```
cluster.module.vm ID -add vm...
cluster.module.vm ID -rm vm...
cluster.module.vm ID # list module VMs
```

Edit: I see that `modules.ls` impl below can list modules or module members, that's fine with me if you prefer   instead of a `module.vm.ls` command.

Reply:
Moved to under cluster